Couchers-org/couchers
# Notifications for community and event discussions

Closes #4240




**Backend checklist**
- [ ] Formatted my code by running `ruff check --select I --fix . && ruff check . && ruff format .` in `app/backend`

**Web frontend checklist**
- [ ] Formatted my code with `yarn format`
- [ ] Added any new components to storybook



Default settings

![Screenshot from 2025-03-08 12-18-36](https://github.com/user-attachments/assets/988119d0-30e6-43ee-b8f2-7d7416f39e1c)

Email templates:


![Screenshot from 2025-03-08 11-30-03](https://github.com/user-attachments/assets/6a270069-c621-46cf-98e6-46d7e3e6dc6a)
![Screenshot from 2025-03-08 11-43-11](https://github.com/user-attachments/assets/da1f3174-816e-4d86-8d15-98f21946a4e9)


## How to test

Enable all these notifs in settings and enable push notifs. Then commenton events that that user is going to, make a discussion in an event they are in, and respond to discussions/comments of that user. Should show up in email as well.
